components/textfield/index.css

```diff
@@ -1289,7 +1291,13 @@ governing permissions and limitations under the License.
 
 	&.spectrum-Textfield--grows {
 		.spectrum-Textfield-input {
-			grid-row: 1 / auto;
+			grid-row: 2 / auto;
+		}
+
+		&.spectrum-Textfield--sideLabel {
+			.spectrum-Textfield-input {
```

Comment:
Might save us a little debugging to un-nest these here since there are no other children for the selector. `&.spectrum-Textfield--sideLabel .spectrum-Textfield-input` might be a touch clearer.